dimod/constrained.py

```diff
@@ -244,6 +244,12 @@ def _add_variables_from(self, model: Union[BinaryQuadraticModel, QuadraticModel]
     def add_constraint(self, data, *args, **kwargs) -> Hashable:
         """A convenience wrapper for other methods that add constraints.
 
+        Examples:
+            >>> from dimod import ConstrainedQuadraticModel, Integers
+            >>> x, y = Integers(['x', 'y'])
```

Comment:
I have tried to follow a convention of `x, y, z` for binary variables, `r, s, t` for spin variables and `i, j, k` for integer. I think it will help in the long run if we're consistent.

Reply:
Updated